src/web/Application.php

```diff
@@ -112,6 +112,13 @@ public function init(): void
         $this->debugBootstrap();
     }
 
+    public function behaviors(): array
+    {
+        return [
+            'corsFilter' => App::corsFilterConfig(),
+        ];
+    }
+
```

Comment:
This couldn't be applied in `src/config/app.web.php` (via `as corsFilter`), because it needs access to `Craft::$app->getConfig()->getGeneral()` for defaults, and the `as {name}` behavior syntax doesn't appear to support a closure.